### ISSUE TITLE
Add Node/NPM Dependency

### ISSUE THREAD

**Issue by adamsdesk** [HappyCerberus/mastodon-scheduler#1]
Add the missing dependency version of Node and NPM to "package.json". I had node v14 installed and didn't realize v16 was minimally required.

```json
{
    "engines": {
        "npm": ">=8.0.0 <9.0.0",
        "node": ">=16.0.0 <17.0.0"
    }
}
```

Refer to official docs, https://docs.npmjs.com/cli/v8/configuring-npm/package-json#engines.

Then enforce the engine dependency by adding ".npmrc" in the same directory as "package.json".

```
engine-strict=true
```

Refer to official docs, https://docs.npmjs.com/cli/v8/configuring-npm/npmrc and https://docs.npmjs.com/cli/v8/using-npm/config#engine-strict.

**Comment by HappyCerberus** [comment]
Thanks for the input. Fixed in 68c4615 and I have also bumped all dependencies to their latest versions ea1f03a.

**Comment by adamsdesk** [comment]
You're welcome!
